pkgs/by-name/st/stripzip-rs/package.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "stripzip-rs";
+  version = "0.0.2";
+
+  src = fetchFromGitHub {
+    owner = "TomaSajt";
+    repo = "stripzip-rs";
+    rev = "v${version}";
+    hash = "sha256-uCHpnvbqFCs3g1ENxEwKspRuTH99QJGimZSo2XPiJGQ=";
+  };
+
+  cargoHash = "sha256-H/oKxRVpX5SIkGpQ0IKkVHpjLXRmGCUuNp9v+wjKQq8=";
+
+  meta = {
+    description = "A tool which removes metadata and unnecessary entries from ZIP archives.";
+    homepage = "https://github.com/TomaSajt/stripzip-rs";
```

Comment:
Nice! But why use this over KittyHawk/stripzip? Perhaps that reasoning could also be in stripzip-rs's README.

Reply:
That tool struggles when there are extra unknown headers on an entry, which causes it to crash. It even has a comment in the code that it's too hard to implement. The rust `zip` package seems to handle it fine.

Reply:
Thanks for adding that detail to the README!